client/makelunch.less

```diff
@@ -0,0 +1,394 @@
+  margin-left: -10px;
+  margin-right: -10px;
+}
+.card {
+  color: #FFF;
+  overflow: hidden;
+  position: relative;
+  margin-bottom: 5px;
+  vertical-align: middle;
+  > .label-top {
+    margin: 0;
+    height: 35px;
+    padding: 10px 10px;
+    width: 100%;
+    background-color: RGBA(250, 73, 23, 1);
+    .score {
+      float: right;
+    }
+  }
+  img { vertical-align: middle; }
+  .stats {
+    transform: translateZ(0);
+    overflow: hidden;
+    position: absolute;
+    text-align: center;
+    top: calc(~"100% - 39px");
+    bottom: 0;
+    font-size: 18px;
+    transition: top 0.15s;
+    padding: 10px;
+    width: 100%;
+    background-color: RGBA(13, 39, 54, 0.8);
+    span {
+      display: block;
+      margin-bottom: 5px;
+    }
+    .card-buttons {
+      position: absolute;
+      left: -100%;
+      width: calc(~"100% - 20px");
+      transition: left 0.15s;
+      bottom: 5px;
+      .btn { display: block; }
+    }
+  }
+  &:hover, &:focus {
+    .stats {
+      border-top: 1px solid white;
+      top: 35px;
+      .card-buttons {
+        left: 10px;
+      }
+    }
+  }
+  img, .photoFrame { width: 100%; }
+}
+/* ADD MEAL */
+body.addmeal, body.editmeal {
+  .card {
+    display: inline-block;
+    width: calc(~"50% - 5px");
+  }
+  .card:nth-child(odd) {
+    margin-right: 5px;
+  }
+  .card .eater-status {
+    position: absolute;
+    width: 100%;
+    top: 35px;
+    bottom: 100%;
+    opacity: 1;
+    transition: all 0.2s;
+  }
+  .card.eating .eater-status {
+    bottom: 0;
+    opacity: 1;
+    background-color: RGBA(13, 39, 54, 0.8);
+    &:before {
+      position: absolute;
+      left: 50%;
+      top: 50%;
+      transform: translate(-50%, -50%);
+      content: '\1F374';
+      color: white;
+      font-size: 64px;
+      text-align: center;
+      text-shadow: 2px 2px 12px rgba(0,0,0,0.8);
+    }
+  }
+  .card.chef .eater-status:before { content: '\1F373'; }
+}
+/* MEALS */
+body.meals .meal {
+  text-align: right;
+  overflow: hidden;
+  position: relative;
+  margin-bottom: 5px;
+  padding: 0 1px;
+  background-color: rgba(255, 255, 255, 0);
+  transition: background-color 0.25s;
+  .photoFrame {
+    display: inline-block;
+    width: 20%;
+    margin: 0 -2px;
+    img {
+      width: 100%;
+      vertical-align: middle;
+    }
+  }
+  .tint {
+    z-index: 2;
+    background-color: RGBA(13, 39, 54, 0.8);
+    transition: background-color 0.25s;
+  }
+  &:hover, &:focus {
+    background-color: RGBA(13, 39, 54, 0.5);
+    .tint {
+      background-color: RGBA(13, 39, 54, 0);
+    }
+    .centred-content {
+      opacity: 0;
+    }
+    .btn-edit-meal {
+      left: 10px;
+    }
+  }
+  .centred-content {
+    opacity: 1;
+    position: absolute;
+    text-align: center;
+    z-index: 3;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    transition: opacity 0.25s;
+    font-size: 20px;
+    color: #FFF;
+    .names {
+      span:not(:last-child):before { content: ' & '; }
```

Comment:
I tend to do that kinda thing as 2 simple selectors.
```css
.names::before {
  content: ' & ';
}
.names:first-child::before {
  content: '';
}
```
That said, in this case your way looks pretty readable. I could be persuaded.